README.md

```diff
@@ -29,6 +29,9 @@ NovelCovid API is recomended by Postman [here](https://covid-19-apis.postman.com
 |  https://corona.lmao.ninja/countries | Returns data of all countries that has COVID-19 |
 |  https://corona.lmao.ninja/countries?sort={parameter} | Returns data of each country sorted by the parameter |
 |  https://corona.lmao.ninja/countries/{country-name} | Returns data of a specific country. If an exact name match is desired pass ?strict=true in the query string |
+|  https://corona.lmao.ninja/countries/{country-Id} | Returns the data of a country if it is equal to its country id  |
```

Comment:
if iso isn't capitalized, then id shouldn't be as well

Reply:
```suggestion
|  https://corona.lmao.ninja/countries/{country-id} | Returns the data of a country if it is equal to its country id  |
```

Reply:
Sorry was fixed